deployer/README.md

```diff
@@ -1175,3 +1175,260 @@ the first created. You can check if the route in question is defined in multiple
     logging     kibana-ops   kibana-ops.example.com              logging-kibana-ops
 
 (In this example there are no overlapping routes.)
+
+## Splitting your Fluentd for high scale capacity (Advanced)
+
+In scenarios where the number of Fluentd nodes starts to exceed 100, the watching
```

Comment:
1. 'Fluentd nodes' - sounds weird to me. 'nodes running Fluentd' i think is better.
2. From what I understand each fluentd starts 2 watches. What's the default value for maxRequestsInFlight? 400?